Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:14.04
+
+COPY ./misc/docker/nginx /etc/nginx/sites-available/nZEDb
+COPY ./misc/docker/init.sh /init.sh
+
+RUN apt-get install -y software-properties-common && \
+  add-apt-repository ppa:ondrej/php5-5.6 && \
+  add-apt-repository ppa:nginx/stable && \
+  apt-get update && \
+  apt-get install -y --force-yes \
```

Comment:
--force-yes may not be needed, but --no-install-recommends helps to keep the image size down